src/main/webapp/populateData.js

```diff
@@ -45,8 +45,14 @@ function populateGmail() {
   const nDays = 7;
   const mHours = 3;
 
+  // Set known values and placeholders while loading
   nDaysContainer.innerText = nDays;
   mHoursContainer.innerText = mHours;
+  unreadEmailsContainer.innerText = '-';
+  unreadEmailsThreeHrsContainer.innerText = '-';
+  importantEmailsContainer.innerText = '-';
+  senderInitialContainer.innerText = '-';
+  senderContainer.innerText = '-';
```

Comment:
I wonder if `...` would be more clear (random suggestion, not really based on any UX data)

Reply:
Agreed! Thanks